SPEC.md

```diff
@@ -430,16 +440,16 @@ Note that the runtime adds the `cniVersion` and `name` fields from configuration
     "interfaces": [
         {
             "name": "cni0",
-            "mac": "00:11:22:33:44:55"
+            "mac": "00:11:22:33:44:55",
```

Comment:
Trailing commas are not valid JSON.

Reply:
Removed trailing commas.  This is where it may make sense to move all JSON and code outside of markdown to separate files and reference the files instead.  Files can be then linted/scanned as part of this build process.